install.sh

```diff
@@ -643,6 +643,11 @@ setup_selinux() {
             $policy_error "Failed to find the k3s-selinux policy, ${policy_hint}"
         fi
     fi
+    # load nft modules for svclb
+    $SUDO modprobe nft-expr-counter || true
+    $SUDO modprobe nfnetlink-subsys-11 || true
+    $SUDO modprobe nft-chain-2-nat || true
```

Comment:
This will load them once, but it will not persist across reboots. This is also in the `setup_selinux` function which doesn't seem like the correct place for this functionality.

Reply:
We should do this in the same place we do the rest of our module loading:
https://github.com/k3s-io/k3s/blob/e64e2fcfd476b4b5669c6845b3a643117f574351/pkg/agent/syssetup/setup.go#L29-L40